cmd/ipfs-cluster-ctl/formatters.go

```diff
@@ -188,7 +188,18 @@ func textFormatPrintPin(obj *api.Pin) {
 		recStr = fmt.Sprintf("Recursive-%d", obj.MaxDepth)
 	}
 
-	fmt.Printf(" | %s\n", recStr)
+	fmt.Printf(" | %s", recStr)
+
+	if len(obj.Metadata) == 0 {
+		fmt.Printf("\n")
+		return
+	}
+
+	fmt.Printf(" | Metadata:")
```

Comment:
Just print `Metadata: yes|no`. Otherwise it just likely that the output will get too polluted. Users can always inspect with --enc=json.